core/arch/x86/instr_create.h

```diff
@@ -2233,6 +2236,29 @@
     instr_create_1dst_2src((dc), OP_kshiftrd, (d), (s1), (s2))
 /* @} */ /* end doxygen group */
 
+/** @name 1 destination, 1 mask and 1 non-immediate source */
+/* @{ */ /* doxygen start group; w/ DISTRIBUTE_GROUP_DOC=YES, one comment suffices. */
+/**
+ * This INSTR_CREATE_mask_xxx macro creates an instr_t with opcode OP_xxx and the given
```

Comment:
s/mask_xxx/ (and the names below) to match the existing style for variations in routines for an opcode: opcode_suffix rather than an infix.

Reply:
Done.